src/assets/_project/_blocks/components/forms/qg-recaptcha.js

```diff
@@ -142,14 +143,14 @@ import keys from '../../data/qg-google-keys';
               response === undefined ||
               response.length === 0
             ) {
-              console.log('Invalid recaptcha');
```

Comment:
why not log?

Reply:
I have added it back, I cleaned that by mistake.